lib/Installation/ModuleRegistration/ModuleRegCodePage.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The module provides interface to act with page that ask for module extra registration code
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::ModuleRegistration::ModuleRegCodePage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub new {
```

Comment:
this method is not needed, inherited from `NavigationBase`.

Reply:
That's interesting. So a base class Perl constructor can call derived class methods ? E.g. I call new() of a base class and it calls its init() method. The init() method called is the one of the derived class I'm creating ?
I'm asking because I saw a constructor defined also on ModuleRegistrationPage, that's not needed too ?

Reply:
yes, it works like that, I think in ruby is the same because the duck typying (no need for interfaces), but for strictly typed language might not be afir.

Reply:
but let's merge, this are details, I also notice that there are other modules that have abbreviations, `transactional => tsm` but we can tackle that later for consistency.